setting.go

```diff
@@ -138,34 +143,56 @@ func (s *ReviewerSet) Entries() []string {
 	return list
 }
 
-func newReviewerSet(list []string) *ReviewerSet {
+func newReviewerSet(list []string, regardAllAsReviewer bool) *ReviewerSet {
```

Comment:
> 別の物ではあるが後から統一したくなったりしないかな？

I don't think so at now.
Then, we should change our code if we'd like to do it!
I seem we need to overhaul the handling configuration before change it.